srcs/runners_manager/vm_creation/VmType.py

```diff
@@ -4,15 +4,13 @@ class VmType:
     """
 
     tags: list[str]
-    flavor: str
-    image: str
+    config: dict
     quantity: dict[str, int]
 
     def __init__(self, config):
         config["tags"].sort()
         self.tags = config["tags"]
-        self.flavor = config["flavor"]
-        self.image = config["image"]
+        self.config = config["config"]
```

Comment:
Exactly what we where talking about. 
100% agreed with this, but we need to edit the `OpenstackManager.py` in this case

Reply:
yeah I pretty much broke the whole code as I didn't adapt the changes at all.